handshake/crypto_setup_client_test.go

```diff
@@ -640,13 +640,15 @@ var _ = Describe("Client Crypto Setup", func() {
 		})
 
 		It("tries to escalate the crypto after receiving a diversification nonce", func(done Done) {
-			go cs.HandleCryptoStream()
-			time.Sleep(50 * time.Millisecond) // wait for the first maybeUpgradeCrypto to finish
+			go func() {
+				defer GinkgoRecover()
+				cs.HandleCryptoStream()
+				Fail("HandleCryptoStream should not have returned")
```

Comment:
Well, if there's no error, then it doesn't return. We have that at a bunch of paces, where we just call `cs.HandleCryptoStream()`. Just here, we're explicit about that.